src/eventHandlers/continuousIntegrationEnd/index.ts

```diff
@@ -141,12 +141,15 @@ const tryMerge = async (
   }
 };
 
-export const checkSuiteHandle = async (
+export const continuousIntegrationEndHandle = async (
   octokit: ReturnType<typeof getOctokit>,
   gitHubLogin: string,
   maximumRetries: number,
 ): Promise<void> => {
-  const pullRequests = context.payload.check_suite.pull_requests as Array<{
+  const pullRequests = (context.eventName === 'workflow_run'
+    ? context.payload.workflow_run
+    : context.payload.check_suite
```

Comment:
I'm not sure how does that work. When I look at the [payload docs](https://docs.github.com/en/free-pro-team@latest/developers/webhooks-and-events/webhook-events-and-payloads#workflow_run), an event for `workflow_run` doesn't have a `workflow_run` property. According to the [trigger docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#workflow_run) it runs against the latest commit to the default (usually `master`/`main`) branch, so I don't know how where would `pull_requests` information come from.

Since it doesn't check out with the docs, we should add a comment why (if) does it work and contribute to GitHub documentation to clarify that.

Reply:
Issue opened https://github.com/github/docs/issues/2599 :-)